docs/user/index.rst

```diff
@@ -0,0 +1,20 @@
+User Guide
+==========
+
+In this user guide, you will find detailed descriptions and
+examples that describe many common tasks that you can accomplish with xarray.
```

Comment:
xarray should be pint

Reply:
Haha yeah, jope @keewis won't mind that I used their documentation structure